src/templates/chat/overkill-reroll.hbs

```diff
@@ -1,7 +1,6 @@
 <div class="card clipped-bot" style="margin: 0px;">
   <div class="lancer-header ">
-    OVERKILL
-    <button class="overkill-reroll"><i class="fas fa-redo"></i> +d6</button>
+    {{{localize "lancer.chat-card.overkill.reroll"}}}
```

Comment:
This part may require some testing or conversion into a handlebar partial template.